ballerina/utils.bal

```diff
@@ -122,133 +83,27 @@ isolated function utcToString(time:Utc utc, string pattern) returns string|error
-}
-
-# Set message attributes to a map of string to add as query parameters.
-#
-# + parameters - Parameter map
-# + attributes - MessageAttribute to convert to a map of string
-isolated function setMessageAttributes(map<string> parameters, MessageAttribute attributes) {
-    int attributeNumber = 1;
-    map<anydata> attributeMap = <map<anydata>>attributes;
-    foreach var [key, value] in attributeMap.entries() {
-        string attributeName = getAttributeName(key);
-        parameters["Attributes.entry." + attributeNumber.toString() + ".Name"] = attributeName.toString();
-        parameters["Attributes.entry." + attributeNumber.toString() + ".Value"] = value.toString();
-        attributeNumber = attributeNumber + 1;
-    }
+isolated function stringToBoolean(string str) returns boolean|error {
+    return check langboolean:fromString(str.toString());
 }
 
-# Set SMS attributes to a map of string to add as query parameters.
-#
-# + parameters - Parameter map
-# + attributes - SmsAttribute to convert to a map of string
-# + return - If successful returns `map<string>` response. Else returns error
-isolated function setSmsAttributes(map<string> parameters, SmsAttributes attributes) returns map<string> {
-    int attributeNumber = 1;
-    map<anydata> attributeMap = <map<anydata>>attributes;
-    foreach var [key, value] in attributeMap.entries() {
-        string attributeName = getAttributeName(key);
-        parameters["Attributes.entry." + attributeNumber.toString() + ".Name"] = attributeName.toString();
-        parameters["Attributes.entry." + attributeNumber.toString() + ".Value"] = value.toString();
-        attributeNumber = attributeNumber + 1;
-    }
-    return parameters;
-}
-
-# Add SMS attributes.
-#
-# + parameters - Parameter map
-# + attributes - Array of attributes to convert to a map of string
-# + return - If successful returns `map<string>` response. Else returns error.
-isolated function addSmsAttributes(map<string> parameters, string[] attributes) returns map<string> {
-    int attributeNumber = 1;
-    foreach var attribute in attributes {
-        parameters["attributes.member." + attributeNumber.toString()] = attribute.toString();
-        attributeNumber = attributeNumber + 1;
-    }
-    return parameters;
-}
-
-# Returns attribute name from field of record. This capitalizes the first letter of the attribute.
-#
-# + attribute - Field name of record
-# + return - If successful returns attribute name string. Else returns error
-isolated function getAttributeName(string attribute) returns string {
-    string firstLetter = attribute.substring(0, 1);
-    string otherLetters = attribute.substring(1);
-    string upperCaseFirstLetter = firstLetter.toUpperAscii();
-    string attributeName = upperCaseFirstLetter + otherLetters;
-    return attributeName;
-}
+isolated function stringToTimestamp(string str) returns time:Civil|error {
+    time:Utc utc = [check stringToInt(str), 0];
+    return time:utcToCivil(utc);
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in [020b0a4](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/020b0a40d31509c3e7b499cf2ea5c620bbfc9b2b)